notebooks/ai-ml/ImageClassificationInSpark.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>training -&gt; inference</p><p> </p><p>Correct link of ResNet50 model <a href="https://pytorch.org/vision/main/models/generated/torchvision.models.resnet50.html" target="_blank">https://pytorch.org/vision/main/models/generated/torchvision.models.resnet50.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>